documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -106,43 +106,43 @@ You might wonder how we're able to use `$page.data` and other [app stores](modul
 <script>
 	import { getContext } from 'svelte';
 
-	// Retrieve user store from context
+	// Récupère le store user depuis le contexte
 	const user = getContext('user');
 </script>
 
-<p>Welcome {$user.name}</p>
+<p>Bienvenue {$user.name}</p>
 ```
 
-Updating the context-based store value in deeper-level pages or components will not affect the value in the parent component when the page is rendered via SSR: The parent component has already been rendered by the time the store value is updated. To avoid values 'flashing' during state updates during hydration, it is generally recommended to pass state down into components rather than up.
+Mettre à jour la valeur d'un store de contexte dans des pages ou composants plus profonds ne va affecter sa valeur dans le composant parent lors de son rendu en <span class="vo">[SSR](PUBLIC_SVELTE_SITE_URL/docs/web#server-side-rendering)</span> : le composant parent a déjà été rendu au moment de la mise à jour de la valeur du store. Pour éviter que les valeurs "clignotent" pendant les mises à jour d'état lors de l'hydratation, il est en général recommandé de faire descendre l'état vers les composants plutôt que de le faire remonter.
```

Comment:
```suggestion
Mettre à jour la valeur d'un store de contexte dans des pages ou composants plus profonds ne va pas affecter sa valeur dans le composant parent lors de son <span class="vo">[rendu côté serveur](PUBLIC_SVELTE_SITE_URL/docs/web#server-side-rendering)</span> : le composant parent a déjà été rendu au moment de la mise à jour de la valeur du store. Pour éviter que les valeurs "clignotent" pendant les mises à jour d'état lors de l'hydratation, il est en général recommandé de faire descendre l'état vers les composants plutôt que de le faire remonter.
```
des fois, je préfère traduire SSR
Ca veut dire quoi : "it is generally recommended to pass state down into components rather than up." ?

Reply:
de le passer en `props` vers des composants plus profonds, plutôt que de binder (binder = remonter l'état)